### PR TITLE
Fix NetperfPackageError

### PR DESCRIPTION
**Pull request by cliping**
Fix following error:
    NetperfPackageError: Packeage Error: Compile failed: Could
    not get exit status of command

Signed-off-by: cliping <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Before:
NetperfPackageError: Packeage Error: Compile failed: Could not get exit status of command: 'cd /var/tmp/  &amp;&amp; tar jxf netperf-2.6.0-1.tar.bz2 &amp;&amp; cd /var/tmp/netperf-2.6.0-1  &amp;&amp; ./autogen.sh &gt; /dev/null 2&gt;&amp;1 &amp;&amp; ./configure --build=x86_64  &gt; /dev/null 2&gt;&amp;1 &amp;&amp; make &gt; /dev/null 2&gt;&amp;1 '    (output: '**  **  **  **  **  **  **  **  **  **  **  **  **  **  **  **  **  **\n                 This System is reserved by ... Recipe Whiteboard: worker\n**  **  **  **  **  **  **  **  **  **  **  **  **  **  **  **  **  **\nLast login: Wed Dec 14 14:07:45 2022 from 10.73.212.18\n')

After:
 (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_vm.positive_testing.live_migration.stress.host.netperf_network.with_postcopy: PASS (278.94 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2022-12-22T21.57-f8e0bec/results.html
JOB TIME   : 280.33 s

**Review by chunfuwen — APPROVED** [review]
LGTM

File: virttest/utils_netperf.py [hunk 1]
```diff
@@ -166,6 +166,7 @@ def pack_compile(self, compile_option=""):
         self.env_cleanup(clean_all=False)
         cmd = "%s && %s " % (pre_setup_cmd, setup_cmd)
         try:
+            self.session.cmd("echo $?", timeout=60)
```
Comment (dzhengfy) [inline virttest/utils_netperf.py:169]: Could you help explain how this change fix the failure？ I could not quite understand. Thanks a lot.

**Review by dzhengfy — APPROVED** [review]
LGTM

File: virttest/utils_netperf.py (hunk 1 above)
Comment (dzhengfy) [reply virttest/utils_netperf.py:169]: It seems a very tricky fix. I accept as it works.

**Pull request merged**